src/db/repositories/user.repository.js

```diff
@@ -8,18 +8,27 @@ const makeUserRepository = () => {
 
   if (!makeUserRepository.db) throw new Error('first you need to call initial method')
 
-  // TODO: implement queries
-  return {
-    getAllUsers: () => {
-
-    },
-    getUserById: async () => {
+  const getUserBy = async (param, value) => {
```

Comment:
In this function we take 2 params, but in other function we take  object. I think it's not predictable and nonuniformity. I think better if params would be object: 
`
{
   <the field search>: <value>
}
`
for example: 
`
{
  email: "user@gmail.com"
}
`